lib/tf_lock/release.py

```diff
@@ -121,32 +121,35 @@ def get_lock_info(root_module: Path) -> Tuple[bool, dict[str, str]]:
     return lock, assert_dict_of_strings(result)
 
 
-def tf_lock_release(root_module: Path, env: Environ) -> None:
+def tf_lock_release(root_module: Path, env: Environ) -> UserError | None:
     lock, lock_info = get_lock_info(root_module)
     if not lock:
         info(f"tf-lock-release: success: {root_module}")
-        return
+        return None
 
     tf_user = f"{get_current_user(env)}@{get_current_host(env)}"
     lock_user = lock_info["Who"]
     if tf_user == lock_user:
         try:
             with sh.cd(tf_working_dir(root_module)):
-                sh.run((
-                    "terraform",
-                    "force-unlock",
-                    "-force",
-                    "--",
-                    lock_info["ID"],
-                ))
+                sh.run(
+                    (
+                        "terraform",
+                        "force-unlock",
+                        "-force",
+                        "--",
+                        lock_info["ID"],
+                    )
+                )
         except sh.CalledProcessError as error:
             # error message was already printed by subcommand
-            raise UserError(code=error.returncode)
+            return UserError(code=error.returncode)
 
         info(f"tf-lock-release: success: {root_module}({lock_user})")
+        return None
```

Comment:
Yea this always prints something, succeed or fail, so I can't understand the blank summary in your test PR comment: https://github.com/getsentry/tacos-gha.demo/pull/2423#issuecomment-2035482202

Reply:
Wait, doesn't it only print if it success? if it fails, it will return the UserError code, unless I'm missing something

Reply:
I think the blank lines are from here https://github.com/getsentry/tacos-gha/blob/835d145cde99c8707fd0d9723e58c27c7fd4f5ed/lib/ci/unlock#L10-L13